workflow-service-sdk/pom.xml

```diff
@@ -27,10 +28,15 @@
             <plugin>
                 <groupId>io.spring.javaformat</groupId>
                 <artifactId>spring-javaformat-maven-plugin</artifactId>
-                <version>0.0.34</version>
-                <configuration>
-                    <skip>true</skip>
-                </configuration>
+                <version>${spring.javaformat.version}</version>
+                <executions>
+                    <execution>
+                        <phase>validate</phase>
```

Comment:
perhaps `apply` is the better option? so we can enforce the formatting of the generated files.
unless we already instruct the generator which formatted to use